lib/docs/data_dictionary.md

```diff
@@ -100,3 +100,25 @@ The column name of the first column contains [embedded metadata](#embedded-metad
 | modifications | List of [modifications](#modifications) |
 | ppm | ppm tolerance |
 | rt_window | Window used for in-source decay correction (min) |
+
+### PGFinder Output
+
+| Field                             | Description                                                                                   |
+|:----------------------------------|:----------------------------------------------------------------------------------------------|
+| `Metadata`                        | Metadata about input files.                                                                   |
+| `ID`                              | Peak identifier.                                                                              |
+| `RT (min)`                        | Retention Time in minutes?                                                                    |
+| `Charge`                          | ?                                                                                             |
+| `Obs (Da)`                        | Observed mass in [Dalton (unit)](https://en.wikipedia.org/wiki/Dalton_(unit))                 |
+| `Theo (Da)`                       | Theoretical mass in  [Dalton (unit)](https://en.wikipedia.org/wiki/Dalton_(unit))             |
+| `Delta ppm`                       | Change in Parts Per Million?                                                                  |
+| `Inferred structure`              | Inferred peptidoglycan structure.                                                             |
+| `Intensity`                       | Intensity of peak in units?                                                                   |
+| `Inferred structure (best match)` | Most likely inferred structure.                                                               |
+| `Intensity (best match)`          | Intensity of the most likely inferred structure.                                              |
+| `Total Intensity`                 | ?                                                                                             |
```

Comment:
Sum of all of the intensities for the structures consolidated. You can use this to compare how much material was actually injected / measured between different runs.